ee/agent/flags/flag_controller.go

```diff
@@ -420,6 +420,14 @@ func (fc *FlagController) OsqueryTlsDistributedWriteEndpoint() string {
 	return fc.cmdLineOpts.OsqueryTlsDistributedWriteEndpoint
 }
 
+func (fc *FlagController) OsqueryVersion() string {
+	return NewStringFlagValue(WithDefaultString("")).get(fc.getControlServerValue(keys.RunningOsqueryVersion))
+}
+
+func (fc *FlagController) SetCurrrentRunningOsqueryVersion(osqueryversion string) error {
```

Comment:
nit - `SetCurrentRunningOsqueryVersion` i think fixing this will fix up some of the failing test mocks

Reply:
fixed typo and added the mocks where it was missing